addons/l10n_hu_edi/views/report_templates.xml

```diff
@@ -2,6 +2,12 @@
 <odoo>
 
     <template id="external_layout_standard" inherit_id="web.external_layout_standard" >
+        <!-- hide company_address as it is taken care of in report_invoice.xml -->
+        <xpath expr="//div[@name='company_address']" position="replace">
+            <t t-if="o and o._name == 'account.move' and o.company_id.account_fiscal_country_id.code == 'HU'"/>
```

Comment:
Could not that be done with a "custom_header2" (or no_header_hu e.g.) then?  I am just afraid that we have other reports on account.move.